README.md

```diff
@@ -131,6 +143,14 @@ scrape_configs:
         regex: .*
         target_label: __address__
         replacement: 127.0.0.1:9115  # Blackbox exporter.
+
+  - job_name: 'blackbox-post'
```

Comment:
This will send to prometheus.io, not the blackbox exporter. There's no need for an example here as it's the save as the above.